### PR TITLE
AD-HOC feat (composer): Add a composer file specification

### PR DESCRIPTION
**Pull request by andrewhowdencom**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "name": "sitewards/webappmanifest",
+    "description": "Web application manifest implementation for Magento",
+    "type": "magento-module",
+    "homepage": "https://github.com/sitewards/magento-webappmanifest.git",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Tobi Hartmann",
```
Comment (toh82) [inline composer.json]: Tobias

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        },
+        {
+            "name": "Andrew Howden",
+            "email": "mail@sitewards.scom",
+            "homepage": "https://www.sitewards.com/",
+            "role": "Developer"
+        }
+    ],
+    "support": {
+        "email": "hello@andrewhowden.com",
```
Comment (toh82) [inline composer.json]: either put in our sitewards adresses above and here or put also here the mail @

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        },
+        {
+            "name": "Patrick Kubica",
+            "email": "mail@sitewards.com",
+            "homepage": "https://www.sitewards.com/",
+            "role": "Developer"
+        },
+        {
+            "name": "Andrew Howden",
+            "email": "mail@sitewards.scom",
```
Comment (toh82) [inline composer.json]: .scom is wrong

**Review by toh82 — CHANGES_REQUESTED** [review]
see comments

File: composer.json (hunk 2 above)
Comment (andrewhowdencom) [reply composer.json]: oh i missed that sorry dudee

**Review by toh82 — APPROVED** [review]
nice

**Pull request merged**